app/messaging/schema/process-application-schema.js

```diff
@@ -33,7 +35,8 @@ const endemicsApplicationSchema = joi.object({
     crn: joi.string().optional(),
     address: joi.string().required(),
     email: joi.string().required().lowercase().email({ tlds: false }),
-    isTest: joi.boolean().optional()
```

Comment:
whats use of this flag?

Reply:
I believe it is for automation tests, but I haven't changed it, I left it as is